spec/namespaces/_core.yaml

```diff
@@ -76,6 +76,57 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/bulk@200'
+  /_bulk/stream:
+    post:
+      operationId: bulk.stream.0
+      x-operation-group: bulk
```

Comment:
The running linter will throw an error on the `operationId` and the `x-operation-group`.
I think you want to do `bulk_stream` for `x-operation-group` and `bulk_stream.0` for `operationId`. Then on the generated client, there will be a `client.bulk_stream()` method, which is different from `client.bulk()`

Reply:
Thanks a lot for the hints, @nhtruong !